lib/pos_mempool.go

```diff
@@ -150,6 +153,23 @@ func NewPosMempool(params *DeSoParams, globalParams *GlobalParamsEntry, readOnly
 	}
 }
 
+func NewPosMempoolNoValidation(params *DeSoParams, globalParams *GlobalParamsEntry, readOnlyLatestBlockView *UtxoView,
```

Comment:
If it's possible to use an instance of the TransactionRegister data structure instead of a virtual mempool, then this constructor won't be needed